src/main/kotlin/app/extractors/PythonExtractor.kt

```diff
@@ -16,11 +16,41 @@ class PythonExtractor : ExtractorInterface {
         }
         val MULTI_IMPORT_TO_LIB =
                 ExtractorInterface.getMultipleImportsToLibraryMap(LANGUAGE_NAME)
+        val COMPREHENSION_MAP = "map"
+        val COMPREHENSION_LIST = "list"
     }
 
     override fun extract(files: List<DiffFile>): List<CommitStats> {
         files.map { file -> file.language = LANGUAGE_NAME }
-        return super.extract(files)
+        val stats = super.extract(files).toMutableList()
+
+        // List comprehension fun fact.
+        val mapRegex = Regex("""(map\([^,]+?,)""")
+        val mapTotalAdded = files.fold(0) { total, diff ->
+            total + diff.getAllAdded().map {
```

Comment:
It's better not to call getAllAdded methods multiple times, suggesting to change code a bit:
```
val allAdded = files.map{ file -> file.getAllAdded() }.flatten()
val allDeleted = files.map{ file -> file.getAllAdded() }.flatten()
...
val mapTotalAdded = allAdded.fold(0) { total, line ->
    total + mapRegex.findAll(line).toList().size }
```